Utilities/DataLoading/EdFi.SmokeTest.Console/Application/SmokeTestsConfiguration.cs

```diff
@@ -132,6 +136,7 @@ public static SmokeTestsConfiguration Create(IConfigurationRoot configuration)
             {
                 ApiUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:ApiUrl")),
                 MetadataUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:MetadataUrl")),
+                XsdMetadataUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:XsdMetadataUrl")),
                 DependenciesUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:DependenciesUrl")),
                 OAuthUrl = configuration.GetValue<string>("OdsApi:OAuthUrl"),
```

Comment:
Since OAuthUrl also contains the instance template, call ResolvedUrl while initializing it (similar to what's done in BulkLoadClientConfiguration.cs)

Reply:
Code changed as requested.